### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by Nyame-Wolf**
# Portfolio: setup and mobile version skeleton
-  Created the headline and nav for my portfolio from microverse-figma template 2
-  Used Figma  to parse a design to create a UI.
-  Used CSS positioning and display flex to style this portfolio project
-  Added images including social media icons to increase appeal of this portfolio
-  checked for and corrected errors using css linters 
-  Corrected code following code reviewer's advice

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            <a href="#"><img src="assets/angellist@2x.png" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="assets/twitter@2x.png" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="assets/medium@2x.png" alt="" /></a>
+          </li>
+        </ul>
+      </div>
```
Comment (zieeco) [inline index.html:60]: **[REQUIRED CHANGES]**

- Please, kindly wrap all these contents in a `section class="headline"` tag as this is the first section in your portfolio to improve web accessibility.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Nyame-Wolf 


Great work on your portfolio 👏 
You've done well implementing the design, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <ul class="link">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li>
+            <a href="#"><img src="assets/Icon-Mail.png" alt="" /></a>
+          </li>
+        </ul>
+      </nav>
+
```
[the remark this answers is not in the document]
Comment (Nyame-Wolf) [reply index.html:29]: Hi, For hiding all links, do I use a media query?

[the remark this answers is not in the document]
Comment (Nyame-Wolf) [reply index.html]: About rectifying code to suit HTML semantic tags, I have done that and pushed to Github. Your review would be highly appreciated.

File: index.html (hunk 1 above)
Comment (Nyame-Wolf) [reply index.html]: I have completed this. Your review would be highly appreciated.

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (Nyame-Wolf) [reply index.html]: also what do you mean by hamburger icon?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      <img id="header-image" src="assets/Illustration-header@2x.png" alt="" />
```
[the remark this answers is not in the document]
Comment (Nyame-Wolf) [reply index.html]: I used the css background property to add the image. I also removed the horizontal scrolling effect.

**Review by topeogunleye — APPROVED** [review]
### Hi @Nyame-Wolf, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**